src/app.ts

```diff
@@ -55,8 +60,21 @@ app.post('/', async (req: Request, res: Response) => {
         .end('Invalid Schema, should be Vega or Vega-Lite.');
   }
 
-  const view = new vega.View(vega.parse(specs), {
+  let loader = vega.loader({ mode: 'http'});
+  loader.http = (uri:string, options:any): Promise<string> => {
+    if (ALLOWED_URLS.every((allowedUrl) => !uri.includes(allowedUrl))) {
```

Comment:
includes is not enough. `http://www.google.com#vega.github.io` if `vega.github.io` is allowed

Reply:
How about `startsWith('vega.github.io')` ?

Reply:
Use https://developer.mozilla.org/en-US/docs/Web/API/URL instead